pasarguard.sh

```
@@ -321,16 +321,36 @@ is_port_in_use() {
     return 1
 }
 
+get_cron_package_name() {
+    if [[ "$OS" == "Ubuntu"* ]] || [[ "$OS" == "Debian"* ]]; then
+        echo "cron"
+    elif [[ "$OS" == "CentOS"* ]] || [[ "$OS" == "AlmaLinux"* ]] || [ "$OS" == "Fedora"* ] || [ "$OS" == "Arch Linux" ]; then
```

Shellcheck (0.11.0)

[error] [ .. ] can't match globs. Use [[ .. ]] or case statement.

(SC2081)